core/templates/components/button-directives/exploration-embed-button-modal.component.spec.ts

```diff
@@ -90,7 +90,7 @@ describe('ExplorationEmbedButtonModalComponent', () => {
     const addRange = jasmine.createSpy('addRange');
     // TS ignore is used here because we are faking the getSelection function
```

Comment:
Please fix -- you're not using TS ignore any more. I suggest you impose the same convention that we use for ts-ignore for explanations of why the check is necessary.

Reply:
done